packages/client/src/client.js

```diff
@@ -117,13 +117,14 @@ export class PercyClient {
   // Creates a build with optional build resources. Only one build can be
   // created at a time per instance so snapshots and build finalization can be
   // done more seemlessly without manually tracking build ids
-  async createBuild({ resources = [] } = {}) {
+  async createBuild({ resources = [], execType } = {}) {
```

Comment:
can you add/update tests for this.

Reply:
There's no update required in tests for `createBuild` as the changes don't include anything logical..